### ISSUE TITLE
Idea: Use lyrics of a website

### ISSUE THREAD

**Issue by ByteHexler** [rakuri255/UltraSinger#13]
I had the idea to create a similar project.
What I thought about was using existing lyrics of a website like genius.com instead of a speech recognition model. There is also an API (https://lyricsgenius.readthedocs.io/en/master/index.html). Alternatetively, you can use the lyrics correct mistakes of the speech recognition model.

**Comment by rakuri255** [comment]
Yeah good idea. I think we need an recognition correction with given lyric.

**Comment by rogerGunis** [comment]
I appreciate this idea - schould give better results in linebreaks and wording
chagpt suggests these web sites for syllables/hypen :-)
https://www.nltk.org/
https://opennlp.apache.org/
https://sapui5.hana.ondemand.com/

**Comment by BanCrash** [comment]
https://github.com/EtienneAb3d/SRT-Sync

```
Synchronize SRT timestamps over an existing accurate transcription

Input 1: SRT with good timestamps and bad-quality text
Input 2: good text-only, or SRT with good text and bad timestamps
Output: SRT with good text and good timestamps
```

**Comment by binarygeek119** [comment]
here a lyric database 
https://lrclib.net/
sample
https://lrclib.net/search/if%20i%20could%20turn%20back%20time%20-%20cher
there are time synced and plain
